opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/_view_instrument_match.py

```diff
@@ -31,21 +31,24 @@
 from opentelemetry.sdk.metrics._internal.view import View
 
 _logger = getLogger(__name__)
-
+OVERFLOW_ATTRIBUTE = ("otel.metric.overflow", "true")
+DEFAULT_AGGREGATION_LIMIT = 2000
 
 class _ViewInstrumentMatch:
     def __init__(
         self,
         view: View,
         instrument: Instrument,
         instrument_class_aggregation: Dict[type, Aggregation],
+        aggregation_cardinality_limit: Optional[int],
```

Comment:
I'm curious as to why `_ViewInstrumentMatch`, `MetricReaderStorage` and `View` all have their separate configurations for `aggregation_cardinality_limit`. Shouldn't this be only configured from `View` and propagated down?